commerce-test-util/src/main/java/com/liferay/commerce/test/util/CommerceTestUtil.java

```diff
@@ -213,6 +236,27 @@ public static CommerceInventoryWarehouse addCommerceInventoryWarehouse(
 				sku, quantity);
 	}
 
+	public static CommerceInventoryWarehouseItem
```

Comment:
@luca-pellizzon I do prefer to have those methods in proper util class and don't mix it